pre_commit/languages/conda.py

```diff
@@ -0,0 +1,64 @@
+import contextlib
+import os
+
+from pre_commit.envcontext import envcontext
+from pre_commit.envcontext import UNSET
+from pre_commit.envcontext import Var
+from pre_commit.languages import helpers
+from pre_commit.util import clean_path_on_failure
+from pre_commit.util import cmd_output_b
+
+ENVIRONMENT_DIR = 'conda'
+get_default_version = helpers.basic_get_default_version
+healthy = helpers.basic_healthy
+
+
+def get_env_patch(env):
+    # On non-windows systems executable live in $CONDA_PREFIX/bin, on Windows
+    # they can be in $CONDA_PREFIX/bin, $CONDA_PREFIX/Library/bin, and
+    # $CONDA_PREFIX/Scripts.
+    path = (os.path.join(env, 'bin'), os.pathsep, Var('PATH'))
+    if os.name == 'nt':  # pragma: no cover (platform specific)
+        path = (env, os.pathsep) + path
+        path = (os.path.join(env, 'Scripts'), os.pathsep) + path
+        path = (os.path.join(env, 'Library', 'bin'), os.pathsep) + path
```

Comment:
this doesn't match the comment, looks like you're adding `env;env/Scripts;env\Library\bin;env\bin` to the path

also this can be
path += (
    os.path.join(env, 'Scripts'), os.pathsep,
    os.path.join(env, 'Library', 'bin'), os.pathsep,
)
```
or some such

the order probably matters here as well?

Reply:
The order matters, thus I cannot do `path +=`. Also I have adjusted the comment as the code is correct. `python.exe` is placed directly in `env`, this took a while to debug to find this problem.